index.js

```diff
@@ -50,12 +50,31 @@ function processProperties (schema, rootSchema, nested, options) {
   return text
 }
 
-function writeDescription (schema) {
-  return `  * ${
-    schema.description || `Represents a ${schema.id} object`
-}
-  * @name ${upperFirst(schema.id)}
-  *
+function writeDescription (schema, options) {
+  const {
+    description = options.autoDescribe === false
+      ? ''
+      : `Represents a${
+        schema.title
+          ? ` ${schema.title}`
+          : 'aeiou'.split('').includes(schema.type.charAt()) ? 'n' : ''
+      } ${schema.type}`
+  } = schema
+
+  const typeMatch = options.types && options.types[schema.type]
+  return `  *${description ? ` ${description}` : ''}
```

Comment:
I understand the intention, but I think there's quite a bit of tenary operators that interfere with the readability especially on the return. Consider splitting/extracting some lines?